inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/yaml/KnowledgeBaseMapping.java

```diff
@@ -177,7 +194,8 @@ public boolean equals(Object o)
                 && Objects.equals(propertyTypeIri, that.propertyTypeIri)
                 && Objects.equals(descriptionIri, that.descriptionIri)
                 && Objects.equals(propertyLabelIri, that.propertyLabelIri)
-                && Objects.equals(propertyDescriptionIri, that.propertyDescriptionIri);
+                && Objects.equals(propertyDescriptionIri, that.propertyDescriptionIri)
+                && Objects.equals(rootConcepts.get(0), that.rootConcepts.get(0));
```

Comment:
Should in principle compare the entire list of root concepts. Btw. *why* does this class need an equals method anyway?

Reply:
Well, if there is no need for the equals (hashcode() as well? because there should never be an equals without a hashcode and vice versa), then I would remove it which saves us the discussion whether or not the entire list needs to be compared ;)

Reply:
Removed